src/components/challengeWidget/challengeWidget.css

```diff
@@ -0,0 +1,32 @@
+.challengeWidget {
+  width: 700px;
+  height: 250px;
+  border: 15px solid burlywood;
+  background-color: white;
+}
+
+h1 {
+  color: black;
+  font-family: sans-serif;
+}
+p {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.first_button {
+  float: right;
+  padding: 5px;
+}
+.paragraph {
+  font-family: serif;
```

Comment:
Let's not override the font face here.

Reply:
I didn't what do you mean by not override the font face?